lib/cocoapods/installer/xcode/pods_project_generator/project_generator.rb

```diff
@@ -0,0 +1,113 @@
+module Pod
+  class Installer
+    class Xcode
+      # Responsible for creating and preparing a Pod::Project instance
+      #
+      class ProjectGenerator
+        # @return [Sandbox] sandbox
+        #         The Pods sandbox instance.
+        #
+        attr_reader :sandbox
+
+        # @return [#to_s] path
```

Comment:
was it `[#to_s]` before? Can be `String`?

Reply:
Yeah `String` is probably better